bundles.rst

```diff
@@ -91,31 +94,31 @@ The directory structure of a bundle is meant to help to keep code consistent
 between all Symfony bundles. It follows a set of conventions, but is flexible
 to be adjusted if needed:
 
-``src/``
-    Contains all PHP classes related to the bundle logic (e.g. ``Controller/RandomController.php``).
+``assets/``
+    Contains the web asset sources like JavaScript and TypeScript files, CSS and
+    Sass files, but also images and other assets related to the bundle that are
+    not in ``public/`` (e.g. Stimulus controllers).
 
 ``config/``
     Houses configuration, including routing configuration (e.g. ``routing.yaml``).
```

Comment:
Wondering if we should set `routes.php` instead? I seems more common for bundle nowday

Reply:
Well i just went through 50% of the bundles listed on the symfony.com dedicated section .... 

... and did not find a single config/routing.php. ... but i did not find more config/routing.yaml ... 

Almost all those bundle are either:
* still on the "old" structure
* not using config files

So i'd say it's even more a good move..  
Because if no one uses/shows this new structure (with routes)... no bundle will follow the trend :)

Reply:
Commited